kuryr_kubernetes/controller/drivers/namespace_security_groups.py

```diff
@@ -87,7 +88,7 @@ def get_security_groups(self, pod, project_id):
 
     def _get_extra_sg(self, namespace):
         # Differentiates between default namespace and the rest
-        if namespace == DEFAULT_NAMESPACE:
+        if namespace in cfg.CONF.namespace_sg.global_namespace:
             return [cfg.CONF.namespace_sg.sg_allow_from_namespaces]
         else:
             return [cfg.CONF.namespace_sg.sg_allow_from_default]
```

Comment:
Perhaps to remove a bit more the usage of `remote_group_id` we could go over the `global_namespaces` and pass along their sgs, instead of using the `sg_allow_from_default`.

Reply:
not sure I follow this... this is for the pods... and it is passing the SG ids. A different thing is how the rules of those SGs look like. And yes, we should definitely remove the remote_group_ids on those 2 SGs. And see how to adapt that for the get_extra_sg on the services side